source/frontend/src/features/documents/list/DocumentListContainer.tsx

```diff
@@ -83,13 +74,8 @@ const DocumentListContainer: React.FunctionComponent<
   };
 
   const updateCallback = useCallback(() => {
-    // Check if the component is working in a File context. If it is, delegate the update.
```

Comment:
@FuriousLlama I believe the reason we were doing this was because of activity documents and file documents being on the same screen so we had to force refresh the whole file when documents was changed. I don't think we need to do that anymore??? hopefully because this is much simpler.

Reply:
Right, this was forcing the update.